openwisp_radius/api/permissions.py

```diff
@@ -25,3 +29,18 @@ def has_permission(self, request, view):
                 _('SMS verification is not enabled for this organization')
             )
         return verification
+
+
+class IsRegistrationEnabled(BasePermission):
+    def has_permission(self, request, view):
+        # check for organization's local setting
+        try:
+            registration_enabled = OrganizationRadiusSettings.objects.get(
```

Comment:
just do: `view.organization.radius_settings.registration_enabled` for consistency with the other views